src/puppetlabs/jdbc_util/core.clj

```diff
@@ -165,3 +165,22 @@
       first
       :count
       pos?))
+
+(defn get-sequence-name
+  "Returns the name of the sequence associated with a column, or nil if there is
+  no sequence."
+  [db table column]
+  (-> (jdbc/query db ["SELECT pg_get_serial_sequence(?, ?)" table column])
+      first
+      :pg_get_serial_sequence))
+
+(defn reconcile-sequence!
```

Comment:
Is this intended to be called eagerly by each service? Or do we expect them to check something first? In the former case, it seems like this should be guarded somehow, so we're not resetting the sequence every time the service starts.